src/main/java/httpServer/RequestParser.java

```diff
@@ -0,0 +1,85 @@
+package httpServer;
+
+import httpServer.http.request.HTTPRequest;
+import httpServer.http.request.HTTPRequestBuilder;
+
+import java.io.IOException;
+
+import static httpServer.http.Constants.CRLF;
+
+public class RequestParser implements RequestParseable {
+
+    public boolean isRequestLine(String line) {
+        if (line.contains(": ")) {
+            return false;
+        }
+        return true;
+    }
+
+    public boolean atEndOfHeaders(StringBuilder processedInput) {
+        return processedInput.toString().contains(CRLF + CRLF);
+    }
+
```

Comment:
If `isRequestLine` and `atEndOfHeaders` are no longer being used outside the class, should they become private?